src/sources/nginx_metrics/mod.rs

```diff
@@ -63,11 +68,13 @@ pub struct NginxMetricsConfig {
     ///
     /// Each endpoint must be a valid HTTP/HTTPS URI pointing to an NGINX instance that has the
     /// `ngx_http_stub_status_module` module enabled.
+    #[configurable(metadata(docs::examples = "http://localhost:8000/basic_status"))]
     endpoints: Vec<String>,
 
-    /// The interval between scrapes, in seconds.
+    /// The interval between scrapes.
     #[serde(default = "default_scrape_interval_secs")]
-    scrape_interval_secs: u64,
+    #[serde_as(as = "serde_with::DurationSeconds<u64>")]
```

Comment:
Noting there's no provided example here, which feels fine to me as showing `5` seconds seems just about as useful as showing `15` 🤷 

Just wanted to point this out in case someone felt differently.

Reply:
I just mirrored the old docs here, which just provided 15s as the default.

Reply:
Yep - general observation on if everything needs examples or if it's fine for "obvious" cases 👍